### PR TITLE
Areej Alharbi

### PR DESCRIPTION
**Pull request by Areejalharbi57**

### CODE REVIEW

**Comment by Areejalharbi57** [comment]
I will compleet it

File: datacorekit/datacorekit/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        }catch {
+            print(error)
+        }
+        tableV.reloadData()
+    }
+    
+    @IBAction func addProduct(_ sender: Any) {
+        
+        var textField = UITextField()
+        var alert = UIAlertController(title: "Alert", message: "Please add new product ", preferredStyle: .alert)
```
Comment (falazmi2022) [inline datacorekit/datacorekit/ViewController.swift]: 1- make alert is 'let' better than 'var

File: datacorekit/datacorekit/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    // MARK: - Core Data stack
+
+    lazy var persistentContainer: NSPersistentContainer = {
+        /*
+         The persistent container for the application. This implementation
+         creates and returns a container, having loaded the store for the
+         application to it. This property is optional since there are legitimate
+         error conditions that could cause the creation of the store to fail.
+        */
+        let container = NSPersistentContainer(name: "datacorekit")
```
Comment (falazmi2022) [inline datacorekit/datacorekit/AppDelegate.swift]: datacorekit you should change this name to same your data model name for work Coredata

**Review by falazmi2022 — COMMENTED** [review]
Great work

**Comment by falazmi2022** [comment]
Also, this repo is for the file system not for Coredate. try  focus so that there are no mistakes in the future

**Pull request closed without merging**